internal/resources/cluster/cluster_spec_flatten_test.go

```diff
@@ -33,8 +33,9 @@ func TestFlattenSpec(t *testing.T) {
 			},
 			expected: []interface{}{
 				map[string]interface{}{
-					clusterGroupKey: "default",
-					proxyNameKey:    "",
+					clusterGroupKey:      "default",
+					proxyNameKey:         "",
+					imageRegistryNameKey: "",
```

Comment:
Was this unit test broken recently? During the last release I noticed all UT's were running fine

Reply:
@vaamarnath unit test was failing, since we are not running it within our CI/CD it was not found. 
On adding unit test as part of CI/CD I had figured it out. 

We will be avoiding such things to happen in future once we have this test running as part of every PR. Done as part of https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/225

@ramya-bangera this change was added after the release.